Extensions/TweenBehavior/tweenruntimebehavior.js

```diff
@@ -418,7 +426,6 @@ gdjs.TweenRuntimeBehavior.prototype.addObjectPositionYTween = function(
 
   this._setupTweenEnding(identifier, destroyObjectWhenFinished);
 };
-
```

Comment:
There are "random" changes like these which seems to move comments out of their original functions